pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -893,6 +899,60 @@ def test_addsub_timedeltalike_non_nano(self, dt64, ts, td):
         assert result._reso == ts._reso
         assert result == expected
 
+    @pytest.mark.xfail(reason="tz_localize not yet implemented for non-nano")
+    def test_addsub_offset(self, ts_tz):
+        # specifically non-Tick offset
+        off = offsets.YearBegin(1)
+        result = ts_tz + off
+
+        assert isinstance(result, Timestamp)
+        assert result._reso == ts_tz._reso
+        # If ts_tz is ever on the last day of the year, the year would be
+        #  incremented by one
+        assert result.year == ts_tz.year
+        assert result.day == 31
+        assert result.month == 12
+        assert tz_compare(result.tz, ts_tz.tz)
+
+    def test_sub_datetimelike_mismatched_reso(self, ts_tz):
+        # case with non-lossy rounding
+        ts = ts_tz
+
+        #  choose a unit for `other` that doesn't match ts_tz's
```

Comment:
Mind expanding on this comment that reso parameterize does test the `>` and `< ` test?